src/sentry/features/manager.py

```diff
@@ -138,24 +138,41 @@ class FeatureManager(RegisteredFeatureManager):
     def __init__(self) -> None:
         super().__init__()
         self._feature_registry: MutableMapping[str, type[Feature]] = {}
+        # Deprecated: Remove entity_features once flagr has been removed.
         self.entity_features: MutableSet[str] = set()
+        self.exposed_features: MutableSet[str] = set()
         self.option_features: MutableSet[str] = set()
         self.flagpole_features: MutableSet[str] = set()
         self._entity_handler: FeatureHandler | None = None
 
-    def all(self, feature_type: type[Feature] = Feature) -> Mapping[str, type[Feature]]:
+    def all(
+        self, feature_type: type[Feature] = Feature, api_expose: bool | None = None
```

Comment:
Nit: Should this be `api_expose: bool = False`?

Reply:
I didn't want to change the default behavior just yet. Getting getsentry and sentry aligned is going to take a few pull requests.

Reply:
I think defaulting it to either `None` or `False` ends up giving the same result though?